model/usersManager.php

```diff
@@ -65,22 +65,40 @@ public function addRecordToUserChallTable($user_id, $challenge_id) {
         ]);
     }
 
-    public function getLeadingTenUsers() {
+    public function getLeadingUsers($how_many) {
         // for the leader board on the HOMEPAGE
         $db = DB::connectDB();
-        $leaders = $db->query(
+        $query =             
             'SELECT 
                 users.username,
                 users.points_total,
-                GROUP_CONCAT(challenges.name) AS challenges
+                GROUP_CONCAT(challenges.id) AS challenge_ids
+                -- GROUP_CONCAT(challenges.name) AS challenges
             FROM users
             LEFT JOIN user_challenge_r
                 ON users.id = user_challenge_r.user_id
             LEFT JOIN challenges
                 ON challenges.id = user_challenge_r.challenge_id
             GROUP BY users.id
-            ORDER BY points_total DESC LIMIT 10;'
-        );
+            ORDER BY points_total DESC LIMIT ' . $how_many;
+        $leaders = $db->query($query);
+
+        // $leaders = $db->prepare(
```

Comment:
Should we just delete this parts of the code since they have been commented out?

Reply:
Thanks! I deleted those lines, please review!